configs/configs_test.go

```diff
@@ -16,30 +16,37 @@ package configs
 
 import (
 	"encoding/json"
+	"fmt"
 	"testing"
 	"time"
 )
 
 func TestJSONDuration(t *testing.T) {
```

Comment:
Maybe `TestJSONDuration_ UnmarshalJSON`, so it's clear which method is being tested?